src/example/dynamic-routing/components/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import MenuItems from "../routes/RouteConfig";
+import { Link } from "react-router-dom";
+import { Param } from "../model/Route";
+
+const Header = () => {
+  const createTo = (path: string, paramObj?: Array<Param>): string => {
```

Comment:
method name should be something meaningful, like generatePathName or getPathName, something like this.